Integrations/integration-Atlassian_Confluence_Server.yml

```diff
@@ -0,0 +1,650 @@
+
+        return res.json()
+
+    def get_content_command():
+        """
+        Confluence Get Content method
+        """
+        space_key = demisto.args().get('key')
+        content_title = demisto.args().get('title')
+
+        raw_content = get_content(space_key, content_title)
+
+        content = {
+            "ID": raw_content['id'],
+            "Title": raw_content['title'],
+            "Type": raw_content['type'],
+            "Version": raw_content['version']['number']
+            "Body": raw_content['body']['view']['value']
+        }
+
+        # create markdown table string from context
+        # the outputs must be array in order the tableToMarkdown to work
+        # headers must be array of strings (which column should appear in the table)
+        md = tableToMarkdown('Content', content, ['ID', 'Title', 'Type'])
+
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': content,
+            'HumanReadable': md,
+            'EntryContext': {
+                'Confluence.Content': content
+            }
+        })
+
+    def search_content(cql,cql_context,expand,start,limit):
+        params = {}
+        #Populate the params dictionary
+        if cql_context is not None:
+            params['cqlcontext'] = cql_context
+        if expand is not None:
+            params['expand'] = expand
+        if start is not None:
+            params['start'] = start
+        params['limit'] = limit
+        params['cql'] = cql
+
+        headers = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json'
+        }
+
+        full_url = BASE_URL + '/content/search'
+
+        res = requests.get(full_url, verify=VERIFY_CERTIFICATE, auth=(USERNAME,PASSWORD), headers=headers, params=params)
+
+        if res.status_code < 200 or res.status_code >= 300:
+            return_error_and_exit('Failed to search Content.\nURL: {}, Status Code: {}, Response: {}'.format(full_url, res.status_code, res.text))
+
+        return res.json()
+
+
+    def search_content_command():
+        """
+        Confluence Search Content method
+        Reference:  https://developer.atlassian.com/server/confluence/advanced-searching-using-cql/
+        """
+
+        cql = demisto.args().get('cql')
+        cql_context = demisto.args().get('cqlcontext')
+        expand = demisto.args().get('expand')
+        start = demisto.args().get('start')
+        limit = demisto.args().get('limit')
+
+        raw_search = search_content(cql,cql_context,expand,start,limit)
+
+        searches = []
+        for result in raw_search['results']:
+            search = {}
+            search['ID'] = result['id']
+            search['Title'] = result['title']
+            search['Type'] = result['type']
+            #search['Version'] = result['version']['number']
```

Comment:
why commented out?

Reply:
I believe it's because it's not a standard field. will remove.